CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.3.3
+ - Changed documentation to clarify use of `replace` config option
```

Comment:
Changelog entries should include a link to the PR(s) that they represent (eg https://github.com/logstash-plugins/logstash-input-s3/blob/master/CHANGELOG.md#337)